examples/linear_model/plot_sparse_logistic_regression_mnist.py

```diff
@@ -0,0 +1,71 @@
+"""
+=====================================================
+MNIST classfification using multinomial logistic + L1
+=====================================================
+
+Performance of multinomial logistic regression with
+L1 penalty. We use the SAGA algorithm for this purpose, which is fast. Test
+accuracy reaches > 0.8, while weight vectors remains *sparse* and
+*interpretable*.
```

Comment:
Note that this accuracy is far below what can be reached by an non-penalized linear model (I think ~0.93) but this should be checked and even more far below the accuracy non linear models such as a multi layer perceptron (0.98+).